src/commands/sourcemaps/__tests__/upload.test.ts

```diff
@@ -16,6 +16,15 @@ describe('upload', () => {
     })
   })
 
+  describe('getMinifiedURL: minifiedPathPrefix is an absolute path', () => {
+    test('should return correct URL', () => {
+      const command = new UploadCommand()
+      command['basePath'] = '/js/sourcemaps'
+      command['minifiedPathPrefix'] = '/js'
+      expect(command['getMinifiedURL']('/js/sourcemaps/common.min.js.map')).toBe('/js/common.min.js.map')
+    })
+  })
```

Comment:
@jybp if the prefix is neither a full URL nor an absolute path, it'll fail in the backend. Would that make sense to add prefix validation directly in the command to give feedback directly when running the command, with a meaningful error message?

Reply:
Good point. I'd argue in favor of having the leading slash optional and letting the backend handle it properly.

Reply:
@lraucy Changed it to check the leading slash.

Should we now make the `--minified-path-prefix` flag optional in the off change no prefix is required?

Reply:
It still make sense to ask for it; `--minified-path-prefix /` says: I want to match everything in this directory as if it was the root directory on my server. I don't think it should be considered as the default behavior, i.e. when the prefix is omitted. Maybe it's worth adding an example with `--minified-path-prefix /` in the documentation though?

Reply:
I updated the readme with a sentence on that. Tell my WDYT